model/autosklearn/dffml_model_autosklearn/config.py

```diff
@@ -106,9 +112,15 @@ class AutoSklearnModelContext(ModelContext):
 
     def __init__(self, parent):
         super().__init__(parent)
-        self.pandas = importlib.import_module("pandas")
-        self.features = self._get_feature_names()
         self._model = None
+        self.pd = importlib.import_module("pandas")
```

Comment:
It was mentioned in the last sync meeting that `importlib` messes with IDE's if you want to use it that's fine. But we don't have to use it.

Reply:
Oh! ok